.rubocop.yml

```diff
@@ -13,41 +13,118 @@ Bundler/OrderedGems:
   Enabled: false
 Gemspec/OrderedDependencies:
   Enabled: false
+Layout/ClosingParenthesisIndentation:
+  Enabled: false
+Layout/FirstArrayElementIndentation:
+  EnforcedStyle: consistent
 Layout/LineLength:
+  Max: 100
+  Exclude:
+    - "spec/**/*_spec.rb"
+Layout/SpaceInLambdaLiteral:
   Enabled: false
-Lint/StructNewOverride:
+Layout/MultilineMethodCallIndentation:
+  Enabled: true
+  EnforcedStyle: indented
+Layout/SpaceInsideHashLiteralBraces:
   Enabled: true
+  EnforcedStyle: no_space
+  EnforcedStyleForEmptyBraces: no_space
+Lint/BooleanSymbol:
+  Enabled: false
+Lint/StructNewOverride:
+  Enabled: false
 Lint/RaiseException:
+  Enabled: false
+Lint/SuppressedException:
+  Exclude:
+    - "spec/spec_helper.rb"
```

Comment:
I actually feel we could remove this one.

We will have valid reason to suppress exceptions from time to time, but those are the sort of situations when I'm very happy to explicitly disable the rubocop rule (which is kind of a way of saying "Yep, I know I'm doing something unusual here, but it's for good reason")